src/confluent_kafka/src/Consumer.c

```diff
@@ -1124,6 +1124,36 @@ Consumer_consumer_group_metadata (Handle *self, PyObject *ignore) {
         return obj; /* Possibly NULL */
 }
 
+static PyObject *
+Consumer_io_event_enable(Handle *self, PyObject *args, PyObject *kwargs)
+{
+    int fd = 0;
+    const char* data = NULL;
+    Py_ssize_t data_size = 0;
+    static char *kws[] = {"fd", "payload", NULL};
+
+	if (!PyArg_ParseTupleAndKeywords(args, kwargs, "iy#", kws,
+                &fd, &data, &data_size))
+    {
```

Comment:
maintain the C-style in this file: open-brace on the same line as if () ..

8-whitespace indentation.